pkg/utils/nodeclass/nodeclass.go

```diff
@@ -97,7 +97,7 @@ func NewSecurityGroupSelectorTerms(securityGroupSelector map[string]string) (ter
 	for k, v := range securityGroupSelector {
 		switch k {
 		case "aws-ids", "aws::ids":
-			ids = strings.Split(strings.Trim(v, " "), ",")
+			ids = lo.Map(strings.Split(v, ","), func(s string, _ int) string { return strings.Trim(s, " ") })
 		default:
 			tags[k] = v
```

Comment:
Based on the changes that you've made, we now need to split the tag value here if it contains a comma